index.js

```diff
@@ -51,10 +51,41 @@ var exec = require('child_process').exec
 var fs = require('fs')
 var accessSync = require('fs-access').sync
 var pkgPath = path.resolve(process.cwd(), './package.json')
+var cordovaPath = path.resolve(process.cwd(), './config.xml')
 var pkg = require(pkgPath)
 var semver = require('semver')
 var util = require('util')
 
+function isCordovaProject () {
```

Comment:
mind pulling these three new methods (isCordovaProject, andriodVersionCode, bumpCordova) into a library file:

`lib/cordova`

I think this would be a good way to divide things up, and would provide a clear path for other people adding new file formats.